Podfile.lock

```diff
@@ -312,22 +312,22 @@ PODS:
     - React-perflogger (= 0.64.0)
   - React-jsinspector (0.64.0)
   - react-native-blur (3.6.1):
-    - React-Core
+    - React
```

Comment:
#16562 introduced `React-Core` instead of `React`, see [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/16562/files#diff-e5b768ab2330ee29074f2eeadddd406f19e73b14673eb5f9b5ed6b5b77f4e363L291-R315).

`React` is the dependency declared [in the library's `.podspec`](https://github.com/Kureev/react-native-blur/blob/9d2d744a5171f3a77564a43f87c2cfb3fbcf597e/react-native-blur.podspec#L17).

Reply:
🤔 after a bunch of different issues all looking like CocoaPods caching stuff, running `bundle exec pod install` reverts the changes from this commit.

I don't understand why the dependency is fine as `React-Core`, though, when [the podspec states it's `React`](https://github.com/Kureev/react-native-blur/blob/9d2d744a5171f3a77564a43f87c2cfb3fbcf597e/react-native-blur.podspec#L17)

@ceyhun any ideas?

Reply:
Also, @AliSoftware, Automattic's CocoaPods expert in residence.

Reply:
@mokagio There's a patch on Gutenberg that converts all `React` to `React-Core` [here](https://github.com/WordPress/gutenberg/blob/7fc4197a6494c040fc3d4e491f22602e058fb126/patches/patch-xcode.js) and the PR introducing that is [here](https://github.com/WordPress/gutenberg/pull/26062).

And for Gutenberg's dependencies in WPiOS we don't actually use the `.podspec`s in the original repos but those in [gutenberg-mobile/third-party-pods](https://github.com/wordpress-mobile/gutenberg-mobile/tree/f9f1f4f0d71be19ff7654d89320102ed82fc90c7/third-party-podspecs) that we generate using [this script](https://github.com/wordpress-mobile/gutenberg-mobile/blob/f9f1f4f0d71be19ff7654d89320102ed82fc90c7/bin/generate-podspecs.sh). You can see that we're pointing to that directory [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/7aa5f25ae0ae97df11f7e4b7072487788a25ab43/Podfile#L155).

In those `.podspec`s you can see that they are indeed `React-Core` for example (https://github.com/wordpress-mobile/gutenberg-mobile/blob/f9f1f4f0d71be19ff7654d89320102ed82fc90c7/third-party-podspecs/react-native-blur.podspec.json#L20).

Reply:
Thank you for the detailed answer @ceyhun 🙇